src/node_report.cc

```diff
@@ -602,17 +602,23 @@ void PrintNativeStack(FILE* fp) {
         IMAGEHLP_LINE64 line;
         line.SizeOfStruct = sizeof(line);
         if (SymGetLineFromAddr64(hProcess, dwAddress, &dwOffset, &line)) {
-          fprintf(fp, "%2d: [pc=0x%p] %s [+%d] in %s: line: %lu\n", i, pSymbol->Address, pSymbol->Name, dwOffset, line.FileName, line.LineNumber);
+          fprintf(fp, "%2d: [pc=0x%p] %s [+%d] in %s: line: %lu\n", i, 
+			reinterpret_cast<void*>(pSymbol->Address), pSymbol->Name, 
+			dwOffset, line.FileName, line.LineNumber);
         } else {
           // SymGetLineFromAddr64() failed, just print the address and symbol
           if (dwOffset64 <= 32) { // sanity check
-            fprintf(fp, "%2d: [pc=0x%p] %s [+%d]\n", i, pSymbol->Address, pSymbol->Name, dwOffset64);
+			  fprintf(fp, "%2d: [pc=0x%p] %s [+%lld]\n", i, 
+				reinterpret_cast<void*>(pSymbol->Address), pSymbol->Name, 
+				dwOffset64);
           } else {
-            fprintf(fp, "%2d: [pc=0x%p]\n", i, pSymbol->Address);
+            fprintf(fp, "%2d: [pc=0x%p]\n", i, 
+			  reinterpret_cast<void*>(pSymbol->Address));
           }
         }
     } else { // SymFromAddr() failed, just print the address
-      fprintf(fp, "%2d: [pc=0x%p]\n", i, pSymbol->Address);
+      fprintf(fp, "%2d: [pc=0x%p]\n", i, 
+		reinterpret_cast<void*>(pSymbol->Address));
```

Comment:
I suspect that line was copied from 4 lines above, where SymFromAddr() has succeeded. The SymFromAddr() call might copy the address into pSymbol->Address in failure cases, but yes, it would be better/more logical to use dwAddress instead at this point. Can you add that to your fix? Thanks.